Lib/idlelib/editor.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, flist=None, filename=None, key=None, root=None):
                 'recent-files.lst')
         self.text_frame = text_frame = Frame(top)
         self.vbar = vbar = Scrollbar(text_frame, name='vbar')
+        self.hbar = hbar = Scrollbar(text_frame, orient="horizontal", name='hbar')
```

Comment:
Use textview.AutoHideScrollbar.  Unless there is a clear reason otherwise, set the command for this and vbar in the initial call.  For the latter, use revised `command=self.handle_yview`.  I don't believe we need to add a `handle_xview`.